features/support/helpers.rb

```diff
@@ -240,3 +240,29 @@ def get_summary_list_rows_by_preceding_heading(heading)
   # some lists have a preceding paragraph
   page.all(:xpath, heading_xpath + dl_rows_xpath).any? ? page.all(:xpath, heading_xpath + dl_rows_xpath) : page.all(:xpath, heading_xpath + paragraph_xpath + dl_rows_xpath)
 end
+
+def is_chrome
+  ENV['CHROME']
+end
+
+def inline_http_requests
+  if is_chrome
+    # Chrome does not support network_traffic, instead we can extract this from the performance logs
+    logs = page.driver.browser.manage.logs.get(:performance)
+    # Store messages in a structure which is easier to work with
+    messages_array = logs.each_with_object([]) do |entry, messages|
+      message = JSON.parse(entry.message)
+      timestamp = entry.timestamp
+      messages << message
+      message.store(:timestamp, timestamp)
+    end
```

Comment:
I don't quite understand what's going on here - could we have a quick chat?

Reply:
@bjgill improved in ad4f126